.travis.yml

```diff
@@ -37,6 +37,10 @@ matrix:
     - php: 5.3
       dist: precise
       env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_VERSION=3.9
+    - php: 5.6
+      env: WP_VERSION=4.2
```

Comment:
This will be overwritten the next time we run `wp scaffold package-tests`.

Rather than run the entire test suite for these versions, can we use a `Scenario Outline` to run the same tests against multiple WP versions?

Reply:
Yes, much better solution, will do.

Reply:
Unfortunately I can't see how to do this currently, without adding a new "Given a WP version X install" type step, so I'll just leave the extra WP versions for the mo, at least until just before merge.

Reply:
Here's how I'd test behavior on a specific WP version:

```
Given a WP install
And I run `wp core download --version=<version> --force`
```

Reply:
Ah very good, will do.